avocado/utils/network/interfaces.py

```diff
@@ -571,3 +571,29 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def are_packets_lost(self, peer_ip, count, option=None, flood=False):
+        """
+        This method is used to check packet loss that occurs during
+        ping to peer address.
```

Comment:
Please, make this [PEP-257](https://peps.python.org/pep-0257/#multi-line-docstrings) compatible.

Reply:
Modified the description to match PEP-257

Reply:
The first line should be smaller, this might break some doc engines like Sphinx during visualization. 

 "Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. "